index.html

```diff
@@ -5,14 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Save the Dinosaur</title>
-    <link rel="stylesheet" href="savethedino.css" />
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
-    <image id="backgroundImage" src="sky_background_mountains.png"> </image> -->
+    <img id="backgroundImage" src="images/background.jpeg">

```

Comment:
`<image>` tag is not a standard HTML tag, it is used for SVG sometimes but I don't think you are using that for this game.
The standard tag for images is `img`